src/SearchResults.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import "./App.css";
+import TableRow from "./TableRow";
+
+const SearchResult = (props) => {
+
+  const dataArray = props.data;
+  return (
+    <table className="table">
+      <thead className="thead-dark">
+        <tr>
+          <th scope="col">ID</th>
```

Comment:
Could maybe have an array of column headers then use map to reduce the numbers of lines here